helm/external-dns-app/templates/deployment.yaml

```diff
@@ -49,11 +49,11 @@ spec:
           - /bin/sh
           - -c
           # GS clusters have the cloud config file in /etc/kubernetes/config/azure.yaml and we can use it as-is so we just copy it to the desired position.
+          # CAPZ clusters use a JSON file so we convert it to yaml and save it to the desired position.
```

Comment:
Still looking good,

It is possible to use multiline like [this](https://github.com/giantswarm/app-updater/commit/2f5102db4f510211f17f842e41065d7b2a20131c#diff-38336ec6220ee6ce281a3b079289ca3eaa21700cb05b3357b893a20c643b24f0R22)